sql-indent-left.el

```diff
@@ -302,6 +307,23 @@ order by xxx desc,
 
 ")
 
+(defun sqlind-indent-select-keywords (syntax base-indentation)
```

Comment:
I think it would be better to just write a new version of `sqlind-indent-select-column` that just aligns the first column to the keyword after SELECT (if such a keyword is present).  This could be placed in `sql-indent.el` and named `sqlind-indent-select-column-alt` (alt is for "alternative", I can't think of a better name that is relatively terse).

You could than just use `sqlind-indent-select-column-alt` only for the `select-clause` syntax symbol instead of both  `sqlind-indent-select-column` and `sqlind-indent-select-keywords`